application/backend/src/main/resources/apps/typerefinery/components/widgets/chart/dialog/.content.json

```diff
@@ -0,0 +1,70 @@
+{
+  "sling:resourceType": "wcm/dialogs/dialog",
+  "tabs": {
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "general": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "chartVariant": {
+        "sling:resourceType": "wcm/dialogs/components/select",
+        "name": "chartVariant",
+        "label": "Choose Graph Variants",
+        "lineChart": {
```

Comment:
please add default option with no value, this way your defaults in the component should work.

Reply:
Ok. It is already handled in code, will remove from here.